### PR TITLE
[SDPT-401] Add 3rd party licenses and update the headers

### PR DESCRIPTION
**Pull request by youminhan**
This is the final version with all the required licenses and headers for OSS approval. 
1. Add a folder named `3rd-party-licenses` which lists all the dependencies and their corresponding license.
2. Add Apache License for this workshop-sample repo.
3. Update the headers for each file that shows the Dell copyright.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ pravega_client_auth_loadDynamic=true
 
 -  You must make the Maven repo in SDP available to your development workstation.
 ```
-kubectl port-forward service/repo 9090:80 --namespace workshop-samples
+kubectl port-forward service/repo 9092:80 --namespace workshop-samples
```
Comment (twincitiesguy) [inline README.md:113]: looks like this accidentally got included here - can we revert this?

File: gradle.properties [hunk 2]
```diff
@@ -17,7 +27,7 @@ logstashLogbackEncoderVersion=4.11
 slf4jApiVersion=1.7.25
 
 # Publishing to project
-projectRepoUrl=http://localhost:9090/maven2
+projectRepoUrl=http://localhost:9092/maven2
```
Comment (twincitiesguy) [inline gradle.properties:30]: looks like there might be a couple more of these unrelated changes.

**Review by twincitiesguy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (youminhan) [reply README.md:113]: Actually, it's a change suggested by Claudio in order to avoid possible conflicts in case the user has standalone Pravega running on the default port 9090. https://github.com/pravega/workshop-samples/pull/4#discussion_r310777684

Comment (twincitiesguy) [reply README.md:113]: I see - actually, i confused this with the code hub changes you made - sorry...  

In that case, this should be fine, although it is an unrelated change, it's small enough we can sneak it in this PR

**Review by twincitiesguy — APPROVED** [review]
